src/Umbraco.Core/PropertyEditors/FileUploadConfigurationEditor.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Umbraco.Cms.Core.IO;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Web.Common.DependencyInjection;
+
+namespace Umbraco.Cms.Core.PropertyEditors
+{
+    /// <summary>
+    /// Represents the configuration editor for the file upload value editor.
+    /// </summary>
+    public class FileUploadConfigurationEditor : ConfigurationEditor<FileUploadConfiguration>
+    {
+        // Scheduled for removal in v12
+        [Obsolete("Please use constructor that takes an IEditorConfigurationParser instead")]
+
+        public FileUploadConfigurationEditor(IIOHelper ioHelper)
+            : this(ioHelper, StaticServiceProvider.Instance.GetRequiredService<IEditorConfigurationParser>())
```

Comment:
```suggestion
        // Scheduled for removal in v12
        [Obsolete("Please use constructor that takes an IEditorConfigurationParser instead")]
        public FileUploadConfigurationEditor(IIOHelper ioHelper)
            : this(ioHelper, StaticServiceProvider.Instance.GetRequiredService<IEditorConfigurationParser>())
```

Additional line break